compose/production/django/Dockerfile

```diff
@@ -70,5 +75,6 @@ COPY --chown=django:django src  /app/src
 COPY --chown=django:django manage.py  /app
 
 USER django
+EXPOSE 8000
```

Comment:
Could you explain what this is for? Why do we need it now and why did we not need it before?

Reply:
It is indeed a leftover (but no harm and probably more standard).

![image](https://user-images.githubusercontent.com/13785185/100023777-dccb3e00-2de5-11eb-99d9-8584054e5183.png)

I added it so docker-slim could work (docker-slim look at the EXPOSE port and probe the api).